### PR TITLE
Update 2020 events pages

### PR DESCRIPTION
**Pull request by kenson2998**
## Types of changes
- [ ] **Bugfix**
- [ ] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [x] **Other (new page prototypes)**

## Description

1. add missed templates and copy template folders from 2019

> ~\src\templates\pycontw-2020\\_includes\portal.html
~\src\templates\pycontw-2020\contents\en\events\overview.html
~\src\templates\pycontw-2020\contents\en\events\sprints.html
~\src\templates\pycontw-2020\contents\zh\events\overview.html
~\src\templates\pycontw-2020\contents\zh\events\sprints.html
~\src\templates\pycontw-2020\events\\
~\src\templates\pycontw-2020\contents_default\

2. edited some 2020 contents on these pages
>~\src\templates\pycontw-2020\\_includes\portal.html
~\src\templates\pycontw-2020\\_includes\menu.html

3. update  src/events/templatetags/events.py   removed **Context()** to:
```py
return template.render({...})
```

## Related Issue
issue #805

**Additional context**
If you want to check the events/Schedule page , you need to create a new Schedule data first .

### CODE REVIEW

**Comment by yychen** [comment]
Looks good to me. But I think you should change the PR description from "Documentation Update" to "Other". I think the documentation update from the template means the documentation in the repository like README.md or DEPLOY.md. This one should be something like make event related pages available, create corresponding templates, or something like that.

Thanks for the effort!

File: src/templates/pycontw-2020/contents/zh/events/sprints.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+			<dt>15:30–18:30</dt>
+			<dd>衝刺</dd>
+			<dt>18:30–19:00</dt>
+			<dd>成果分享</dd>
+		</dl>
+
+		<h3>有哪些專案？</h3>
+		<dl>
+			<dt><a href="https://github.com/pycontw/pycontw2016">PyCon Taiwan 官方網站</a></dt>
+			<dd>一起打造 PyCon Taiwan 2018 的網站吧！</dd>
```
Comment (tai271828) [inline src/templates/pycontw-2020/contents/zh/events/sprints.html:89]: 2018 --> 2020

File: src/templates/pycontw-2020/events/schedule.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+</script>
+
+{% block controls %}{% endblock controls %}
+
+{% endblock content %}
+
+
+{% block scripts %}
+{{ block.super }}
+<script type="module" src="{% static 'pycontw-2018/scripts/schedule.js' %}"></script>
```
Comment (tai271828) [inline src/templates/pycontw-2020/events/schedule.html:70]: 2018 --> 2020

File: src/templates/pycontw-2020/_includes/portal.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+
+<div class="card">
+	<a class="button-round" href="{% url 'events_schedule' %}" class="card-link">{% trans 'Schedule' %}</a>
+</div>
+
+<div class="card">
+	<a class="button-round" href="https://hackmd.io/@PyConTW/2019" class="card-link">{% trans 'Collaborative Notes' %}</a>
```
Comment (tai271828) [inline src/templates/pycontw-2020/_includes/portal.html:8]: 2019 --> 2020

File: src/templates/pycontw-2020/_includes/portal.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+</div>
+
+<div class="card">
+	<a class="button-round" href="http://t.me/pycontw2018" class="card-link">{% trans 'Chat Room' %}</a>
```
Comment (tai271828) [inline src/templates/pycontw-2020/_includes/portal.html:12]: Currently it is `http://t.me/pycontw2018` but we are going to use `http://t.me/pycontw`

File: src/templates/pycontw-2020/contents/en/events/sprints.html [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+			<dt>15:30–18:30</dt>
+			<dd>Sprinting</dd>
+			<dt>18:30–19:00</dt>
+			<dd>Sharing / Summary</dd>
+		</dl>
+
+		<h3>What Projects Are There?</h3>
+		<dl>
+			<dt><a href="https://github.com/pycontw/pycontw2016" target="_blank" rel="noopener">PyCon Taiwan’s Official Website</a></dt>
+			<dd>Let’s build PyCon Taiwan 2018’s website together!</dd>
```
Comment (tai271828) [inline src/templates/pycontw-2020/contents/en/events/sprints.html:89]: ditto

File: src/templates/pycontw-2020/_includes/portal.html (hunk 4 above)
Comment (tai271828) [reply src/templates/pycontw-2020/_includes/portal.html:12]: We could keep this link until we do update the link of telegram. Please keep contacting with me.

**Comment by tai271828** [comment]
@Lee-W please help us review the program-related contents like "if we are going to have sprint events".

File: src/templates/pycontw-2020/_includes/portal.html (hunk 3 above)
Comment (kenson2998) [reply src/templates/pycontw-2020/_includes/portal.html:8]: I'll edit it ,by the way ,this page is not found now ~

File: src/templates/pycontw-2020/_includes/portal.html (hunk 4 above)
Comment (kenson2998) [reply src/templates/pycontw-2020/_includes/portal.html:12]: I got it , I though `http://t.me/pycontw2018` is working , so I didn't edit it , okay I'll update to  `http://t.me/pycontw` .

Comment (tai271828) [reply src/templates/pycontw-2020/_includes/portal.html:12]: No. Sorry for confusing. Please keep the original link and I will update later when it's the link migration date.

**Comment by tai271828** [comment]
According to the discussion on telegram, we will merge this pull request first as a basic structure, then merge the other following pull requests for real contents (provided by the program team), and then deploying.

**Pull request merged**